README.md

```diff
@@ -220,6 +221,20 @@ Options:
   - ```-dg, --datagroup <datagroup>``` - Name of the existing datagroup to assign to the published packages
   - ```-?, -h, --help``` - Show help and usage information
 
+### undeploy
+
+Creates a new CustomerEnvironment's version and terminates the other versions, removing deployments
+
+Equivalent to powershell cmdlet <a href="#reset-environment">Reset-Environment</a>
+
+Usage: ```cmf-portal publish-package [options]```
+
+Options:
+  - ```-v, --verbose``` - Show detailed logging
+  - ```-n, --name <name>``` - **REQUIRED** - Name of the Customer Environment to be used
+  - ```-tovrv, --terminateOtherVersionsRemoveVolumes``` - Flag that controls if the volumes of the versions of the Customer Environment that will be terminated should be removed. Requires the terminateOtherVersionsRemove flag.
```

Comment:
Looks like you removed the "terminateOtherVersionsRemove" flag. Any reason why we should not have the additional functionality with the flag? It may not be our use case but why not just leave the flag?
```suggestion
  - ```-tovrv, --terminateOtherVersionsRemoveVolumes``` - Flag that controls if the volumes of the versions of the Customer Environment that will be terminated should be removed.
```

Reply:
Ok. Nonetheless there is no terminateOtherVersionsRemove flag for this command so the last sentence can be removed.